### PR TITLE
Dapr/release/ch1 world dist

### PR DESCRIPTION
**Pull request by robvet**
PR for Chapter 1 for Dapr for .NET Developers eBook

### CODE REVIEW

File: docs/architecture/dapr-for-net-developers/the-world-is-distributed.md [hunk 1]
```diff
@@ -1,75 +1,93 @@
...
 - What are the costs?
 - And, importantly, what are the tradeoffs?
 
-For years, we designed, developed, and deployed applications as a single, monolithic unit. Figure 1.x shows a monolithic architecture.
+To start, let's rewind and look at the past 15 years. During this period, we typically constructed applications as a single, monolithic unit. Figure 1-1 shows the architecture.
 
 ![Monolithic architecture.](./media/monolithic-design.png)
 
-**Figure 1**. Monolithic architecture.
+**Figure 1-1. Monolithic architecture.
```
Comment (sughosneo) [inline docs/architecture/dapr-for-net-developers/the-world-is-distributed.md:25]:
```suggestion
**Figure 1-1** : Monolithic architecture.
```

File: docs/architecture/dapr-for-net-developers/the-world-is-distributed.md [hunk 2]
```diff
@@ -1,75 +1,93 @@
...
 
-Many organizations are mitigating the monolithic fear cycle. They are gaining speed and agility by adopting a distributed architectural approach to building systems. Figure 1 shows the same system built applying cloud-native techniques and practices.
+Instead of fear, businesses require `speed and agility`. They seek an architectural style with which they can rapidly respond to market conditions. They need to instantaneously update and individually scale small areas of a live application.
+
+An early attempt to gain speed and agility came in the form of [Service Oriented Architecture](https://en.wikipedia.org/wiki/Service-oriented_architecture), or `SOA`. In this model, service consumers and service providers collaborated via middleware messaging components, often referred to as an [Enterprise Service Bus](https://en.wikipedia.org/wiki/Enterprise_service_bus), or `ESB`. Figure 1-2 shows the architecture.
+
+![SOA.](./media/soa-basic.png)
+
+**Figure 1-2. SOA architecture.
```
Comment (sughosneo) [inline docs/architecture/dapr-for-net-developers/the-world-is-distributed.md:59]:
```suggestion
**Figure 1-2** : SOA architecture.
```

File: docs/architecture/dapr-for-net-developers/the-world-is-distributed.md [hunk 3]
```diff
@@ -1,75 +1,93 @@
...
-- Each release as small as possible and requires a full deployment of the entire application.
-- One unstable component can crash the entire system.
-- New technologies and frameworks aren't an option.
-- It's difficult to implement agile delivery methodologies.
-- Architectural erosion sets in as the code base deteriorates with never-ending "special cases."
-- The consultants tell you to rewrite it.
+ - The monolith has become so overwhelmingly complicated that no single person understands it.
+ - You fear making changes as each brings unintended and costly side effects.
+ - New features/fixes become time-consuming and expensive to implement. 
+ - Even the smallest change requires a full deployment of the entire application - expensive and risky.
```
Comment (sughosneo) [inline docs/architecture/dapr-for-net-developers/the-world-is-distributed.md:44]:
```suggestion
 - Even the smallest change requires full deployment of the entire application - expensive and risky. 
```

File: docs/architecture/dapr-for-net-developers/the-world-is-distributed.md [hunk 4]
```diff
@@ -1,75 +1,93 @@
...
+ 
+For each of these challenges, multiple products are often available. But, shielding your application from product differences and keeping code maintainable and portable become a challenge.
 
-| Challenge | Description |
-| :-------- | :-------- |
-| State Management | Maintaining contextual information during a transaction |
-| Interservice Communication | Communicate with other independent services |
-| Messaging |  |
-| Obseraviblity | End-to-end monitoring of processes executing on different machines |
+This book introduces Dapr. Dapr is a distributed application runtime. It directly addresses many of the challenges found that come along with distributed applications. Looking ahead, Dapr has the potential to have profound impact on distributed application development.
```
Comment (sughosneo) [inline docs/architecture/dapr-for-net-developers/the-world-is-distributed.md:84]:
```suggestion
This book introduces Dapr. Dapr is a distributed application runtime. It directly addresses many of the challenges found that come along with distributed applications. Looking ahead, Dapr has the potential to have a profound impact on distributed application development.
```

File: docs/architecture/dapr-for-net-developers/the-world-is-distributed.md [hunk 5]
```diff
@@ -1,75 +1,93 @@
...
+With SOA, centralized service providers registered with the ESB. Business logic would be built into the ESB to integrate providers and consumers. Service consumers could then find and communicate with these providers using the ESB. 
+
+Despite the promises of SOA, implementing this approach often increased complexity and introduced bottlenecks. Maintenance costs became high and ESB middleware expensive. Services tended to be large. They often shared dependencies and data storage. In the end, SOAs often resulted in a 'distributed monolithic' structure with centralized services that were resistant to change. 
+
+Nowadays, many organizations have realized speed and agility by adopting a distributed microservice architectural approach to building systems. Figure 1-3 shows the same system built using distributed techniques and practices.
 
 ![Distributed architecture.](./media/distributed-design.png)
 
-**Figure 1**. Distributed architecture.
+**Figure 1-3. Distributed architecture.
```
Comment (sughosneo) [inline docs/architecture/dapr-for-net-developers/the-world-is-distributed.md:69]:
```suggestion
**Figure 1-3** : Distributed architecture.
```

**Comment by sughosneo** [comment]
Hi @robvet - This PR has some lint errors. If you could review. 
I have also included a few suggestions.

Thank you.

File: docs/architecture/dapr-for-net-developers/the-world-is-distributed.md (hunk 2 above)
Comment (robvet) [reply docs/architecture/dapr-for-net-developers/the-world-is-distributed.md:59]: Great catch, thx.

File: docs/architecture/dapr-for-net-developers/the-world-is-distributed.md (hunk 4 above)
Comment (robvet) [reply docs/architecture/dapr-for-net-developers/the-world-is-distributed.md:84]: Done

**Review by nishanil — APPROVED** [review]

**Pull request merged**